phpunit.mongo.xml

```diff
@@ -13,7 +13,7 @@
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak_vendors" />
         <server name="KERNEL_DIR" value="tests/Fixtures/app/" />
         <server name="KERNEL_CLASS" value="AppKernel" />
-        <server name="APP_ENV" value="test_mongodb" />
+        <server name="APP_ENV" value="mongodb" />
```

Comment:
Are you sure it's working as expected? Because there is a specific configuration for this environment:
- https://github.com/api-platform/core/blob/master/tests/Fixtures/app/config/config_test_mongodb.yml
- https://github.com/api-platform/core/blob/master/tests/Fixtures/app/config/routing_test_mongodb.yml

Reply:
Well, tests are green so you tell me 😄

Reply:
@Toflar But the config wouldn't get applied correctly. So the `APP_ENV` should be reverted to `test_mongodb`. :smile:

Reply:
Or rename the config files.

Reply:
@Toflar I think we should rename the config files instead.

Reply:
I don't understand. Can you point me to where that's done for elasticsearch? I haven't checked that testsuite yet.

Reply:
@alanpoulain Can you advise why there is a `config_orm.yml` (imported from `config_test.yml`) and `config_mongodb.yml` (I don't see where this is imported)?

Reply:
Note https://github.com/api-platform/core/blob/v2.4.2/tests/Fixtures/app/AppKernel.php#L97

It's very confusing if we name config files the same way when it's not environment-specific.

Reply:
Okay @Toflar, so my suggestion is that we should merge `mongodb` and `test_mongodb` environments into one `mongodb` environment.

Reply:
Yeah that's what I was trying 😄 So I revert the latest commit again?

Reply:
Yes... But we need to make sure the relevant parts of the old `test_mongodb` environment are merged in, e.g.:

tests/Fixtures/app/config/config_test_mongodb.yml
tests/Fixtures/app/config/config_services_mongodb.yml
tests/Fixtures/app/config/routing_test_mongodb.yml

And the old files removed.